docs/src/index.md

```diff
@@ -4,7 +4,33 @@ CurrentModule = Taxonomy
 
 # Taxonomy
 
-Documentation for [Taxonomy](https://github.com/StructuralEquationModels/Taxonomy.jl).
+`Taxonomy.jl` is work in progress but here is our vision about what it is supposed to do when finished:
+
+## What?
+
+Taxonomy.jl aims to serve as a comprehensive database of structural equation models (SEMs) that can be used to infer distributions of both structures and parameters.
+This will greatly facilitate simulation studies that accurately reflect real-world conditions.
```

Comment:
Maybe we could shortly elaborate here, on why this is naccessary ( maybe with citation of Pawel et al.? https://arxiv.org/pdf/2203.13076.pdf

I think for some researchers, especially when they are longer in the game this might be important to see the relevance of this project more clearly - especially in contrast to other tools/ workflows they might be using

Reply:
well-defined study aims and methodology are as important in simulation studies as in non-simulation based studies because the opportunities of possible designs and analysis are much larger (Hoffmann et al., 2021). Therefore reserchers should define: the study goals, the data-generating process, the methods under investigation, the estimands of interest, the evaluation metrics, or how missing values should be handled, otherwise there would be to much room for inaccuracy (Simmons et al., 2011). With simulated data, researchers would be free to then generate a multiplicity of possible results which might foster overoptimistic impressions if they would report only the subset of results aligning with their hypotheses and beliefs, and for which they can find plausible explanations post hoc.

#maybe something like this - see pawel et al. page 5

Reply:
```suggestion
This will greatly facilitate simulation studies that accurately reflect real-world conditions and takes seriously the idea that "Simulation studies are to a statistician what experiments are to a scientist" [Pawel & Kook et al.](https://doi.org/10.1002/bimj.202200091).
```